src/app/fyle/merge-expense/merge-expense-3.page.spec.ts

```
@@ -85,7 +84,7 @@ export function TestCases3(getTestBed) {
       dependantFieldsService = TestBed.inject(DependentFieldsService) as jasmine.SpyObj<DependentFieldsService>;
       formBuilder = TestBed.inject(UntypedFormBuilder);
       component.fg = formBuilder.group({
-        target_txn_id: [, Validators.required],
+        target_expense_id: [, Validators.required],
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)